### DISCUSSION TITLE
Can I use Mapsui? WinUI + toggle different data layer :)

### DISCUSSION THREAD

**Discussion by ghost** [Mapsui/Mapsui#1953] [Q&A] [answered]
I have some questions if I can use Mapsui :). 

Maybe related to the ‘Supported data’ section [1].

I have a future project in WinUI that includes map functionality. It should be possible to interact with the map SDK/API and WinUI. 

1)	The user can choose which data layer to display. For example, it may be that images need to be displayed. Or different values/features per coordinate. The most important thing is to be able to display different layers of data. 

2)	 Openstreetmap + Mapsui could be interesting?

[1] https://mapsui.com/documentation/home.html#supported-data

**Comment by pauldendulk** [comment, marked as answer]
The Map component has a list of layers and you can control from code which layers to show or hide. It is possible to use the openstreetmap tile layer, it is used on most of the samples. It is possible to draw points, lines, polygon and raster images. Mapsui does not have a lot of data formats it can read, but it is often possible to write your IProvider for some data format. Does that answer you question?

**Reply by ghost** [reply]
Yes! Thank you very much!
